stratum/glue/BUILD

```diff
@@ -57,3 +59,9 @@ stratum_cc_library(
         "integral_types.h",
     ],
 )
+
+cc_library(
+    name = "stamping",
```

Comment:
Nit: I think `version_stamp.h` and `version_stamp.cc` might be a bit more descriptive.
Or, `version.h` and `version_linkstamp.cc`

What do you think?

Reply:
I had `version.h` in the draft, but though that's too limited because we also do time, host and user besides git ref (what I consider our version).

No strong feelings either way, though.